player.py

```diff
@@ -259,3 +275,58 @@ def take_food_from_trail(self, trail, ant_positions, ant, direction):
     trail[food_position] = None
 
     return (food_to_hand, trail)
+
+  def move_ant(self, trail, ant_positions, anthill, ant):
+    """Asseses the layout and calls the appropriate method to advance the insect
+
+    Has control flow to account for ants which are: 
+      -not yet on the trail moving onto the trail
+      -already on the trail and moving further along the trail
+      -on the trail and moving past the end onto the anthill
+
+    Parameters
+    ----------
+    trail : (list)
+      The trail contains a list of the food tokens available on the game area.
+      The types of food token can be found in constants.py/FOOD_TYPES
+      As food tokens are removed from the game the elements are replaced with None
+
+    ant_positions : (dict)
+      A dictionary showing the starting location of each ant.
+      The position on the trail will be defined as the index of the 
+      element in the trail list.
+      Keys are strings
+      Values are integers
+
+    anthill : (list)
+      A list with the same length as the number of ants in the game. 
+      Initialised with each element as None.
+      Elements will be changed into the IDs of the ants as they reach the anthill.
+      Each element is None or string.
+
+    ant : (string)
+      The ID of the ant which is being placed onto the anthill.
+
+    Returns
+    -------
```

Comment:
Technically, this returns `return_tuple`, which in turn contains antill and ant positions